clang/include/clang/Driver/Options.td

```diff
@@ -2986,6 +2986,14 @@ def fsycl_link_EQ : Joined<["-"], "fsycl-link=">,
 def fsycl_link : Flag<["-"], "fsycl-link">, Alias<fsycl_link_EQ>,
   AliasArgs<["early"]>, Flags<[CC1Option, CoreOption]>,
   HelpText<"Generate partially linked device object to be used with the host link">;
+def fsycl_header_only_library : Flag<["-"], "fsycl-header-only-library">,
+  Group<f_Group>, Flags<[CC1Option, NoArgumentUnused]>,
+  HelpText<"Tells the SYCL host compilation not to implicitly add -lsycl and "
+           "look for a SYCL library as we will use a header only runtime "
+           "library.">;
+def fsycl_mutable_global : Flag<["-"], "fsycl-mutable-global">,
+  Flags<[CC1Option, CoreOption]>, HelpText<"Allow mutable global variables in SYCL device code">,
+  MarshallingInfoFlag<LangOpts<"SYCLAllowMutableGlobal">>;
```

Comment:
How is this different from Intel extension for device-global variable?
https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/proposed/sycl_ext_oneapi_device_global.asciidoc

Reply:
The link seems dead